libs/live-wallet/src/store.test.ts

```diff
@@ -193,20 +194,24 @@ describe("Wallet store", () => {
   it("can export the wallet state", () => {
     const exportedState = {
       walletSyncState: { data: {}, version: 42 },
+      nonImportedAccountInfos: [],
```

Comment:
maybe move it outside and then you can do 

expect(exportWalletState(result)).toEqual(exportedState);